templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -25,6 +25,48 @@
 			{{end}}
 		</div>
 
+
+		<div class="ui divider"></div>
+
+		<div class="ui {{if or (not .IsIssueWriter) .Repository.IsArchived}}disabled{{end}} floating jump select-project dropdown">
+			<span class="text">
+				<strong>{{.i18n.Tr "repo.issues.new.projects"}}</strong>
+				<span class="octicon octicon-gear"></span>
+			</span>
+			<div class="menu" data-action="update" data-issue-id="{{$.Issue.ID}}" data-update-url="{{$.RepoLink}}/issues/projects">
+				<div class="no-select item">{{.i18n.Tr "repo.issues.new.clear_projects"}}</div>
+				{{if .OpenProjects}}
+					<div class="divider"></div>
+					<div class="header">
+						<i class="octicon octicon-milestone"></i>
+						{{.i18n.Tr "repo.issues.new.open_projects"}}
+					</div>
+					{{range .OpenProjects}}
+						<div class="item" data-id="{{.ID}}" data-href="{{$.RepoLink}}/issues?project={{.ID}}"> {{.Title}}</div>
+					{{end}}
+				{{end}}
+				{{if .ClosedProjects}}
+					<div class="divider"></div>
+					<div class="header">
+						<i class="octicon octicon-milestone"></i>
```

Comment:
may we can change to new system for octicons like this?

Reply:
Yes, this will no longer work, need to do `{{svg "octicon-milestone" 16}}` now.